extension/demo/src/modules/storage.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+import browser from 'webextension-polyfill'
+
+export async function set (data) {
+  await browser.storage.local.set(data)
+
+  return !browser.runtime.lastError
+}
+
+export function get (key) {
+  return new Promise(async resolve => {
+    const results = await browser.storage.local.get([key])
+    resolve(results[key])
+  })
+}
```

Comment:
```suggestion
export async function get (key) {
  const results = await browser.storage.local.get([key])
  return results[key]
}
```

Reply:
This looks like the code that we had which did not work but we can try again

Reply:
Sounds strange. What wasn't working?